airbyte-cdk/python/unit_tests/sources/file_based/test_scenarios.py

```diff
@@ -105,6 +105,7 @@ def _verify_read_output(output: EntrypointOutput, scenario: TestScenario[Abstrac
 
     expected_states = list(filter(lambda e: "data" not in e, expected_records))
     states = list(filter(lambda r: r.state, records))
+    assert len(states) != 0, "No state messages were emitted."
```

Comment:
nit can we make this error: "No state messages emitted. Successful syncs should emit at least one stream state."

Reply:
```suggestion
    assert len(states) > 0, "No state messages emitted. Successful syncs should emit at least one stream state."
```